react/src/components/dropdown/SprkDropdown.js

```diff
@@ -165,7 +188,6 @@ class SprkDropdown extends Component {
                   isActive,
                   text,
                   value,
```

Comment:
why'd we remove idString here?

Reply:
I assume it was removed because it causes an eslint error, but we should definitely put it back. With this removed `data-id` is pulling the wrong values on the options.